scripts/popup.js

```diff
@@ -484,7 +504,23 @@ chrome.runtime.onMessage.addListener(
   }
 )
 
-window.onloadFuncs = [checkExcluded, borrow_books, show_news, show_wikibooks, search_box_activate, noContextTip, setupWaybackCount]
+function emailSharing() {
+  var sharing_url = ''
+  var overview_url = 'https://web.archive.org/web/*/'
+  chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+    let url = tabs[0].url
+    if (isNotExcludedUrl(url)) { // Prevents sharing some unnecessary page
+      if (url.includes('web.archive.org')) {
+        sharing_url = url // If the user is already at a playback page,share that URL
+      } else {
+        sharing_url = overview_url + get_clean_url(url) // When not on a playback page,share the overview version of that URL
+      }
+      $('#email-share').attr('href', `mailto:?subject=Hey, please check out this amazing website.&body=${sharing_url}`)
```

Comment:
You can change the subject here like "Hey, Please checkout all archives of this website available in web.archive.org"

Reply:
Sure, and, what about the "Toast" messages? Please tell me if you want any changes there :)

Reply:
Looks fine to me !